### PR TITLE
feat(inbox): Revised two step inbox guide

### PR DESCRIPTION
**Pull request by taylangocmen**
This removes the dynamic counts guide and makes the inbox guide the default guide for forcing assistant. It also splits the guide into two steps as described in the Jira card.

See the [Jira card](https://getsentry.atlassian.net/browse/WOR-472) for specs

### CODE REVIEW

File: src/sentry/static/sentry/app/stores/guideStore.tsx [hunk 1]
```diff
@@ -194,10 +194,10 @@ const guideStoreConfig: Reflux.StoreDefinition & GuideStoreInterface = {
     const userDateJoined = new Date(user?.dateJoined);
 
     if (!forceShow) {
-      guideOptions = guideOptions.filter(({guide, seen}) =>
+      guideOptions = guideOptions.filter(({/*guide, */ seen}) =>
```
Comment (davidenwang) [inline src/sentry/static/sentry/app/stores/guideStore.tsx:197]: are these comments still intended?

File: src/sentry/static/sentry/app/views/issueList/header.tsx [hunk 2]
```diff
@@ -119,35 +119,40 @@ function IssueListHeader({
...
+
+            return (
+              <li key={tabQuery} className={query === tabQuery ? 'active' : ''}>
+                <Link to={to}>
+                  <GuideAnchor
+                    target={inboxGuideStepOne ? 'inbox_guide_tab' : 'none'}
+                    disabled={!inboxGuideStepOne}
+                    to={to}
+                  >
+                    <GuideAnchor
```
Comment (scttcper) [inline src/sentry/static/sentry/app/views/issueList/header.tsx:138]: might look at not rendering these somehow. like a function that takes children and either wraps it or it doesn't

**Review by scttcper — APPROVED** [review]

File: src/sentry/static/sentry/app/stores/guideStore.tsx (hunk 1 above)
Comment (taylangocmen) [reply src/sentry/static/sentry/app/stores/guideStore.tsx:197]: oops missed this when pick adding

File: src/sentry/static/sentry/app/views/issueList/header.tsx (hunk 2 above)
Comment (taylangocmen) [reply src/sentry/static/sentry/app/views/issueList/header.tsx:138]: I agree but that also looked a little convoluted, disabling this guides seemed to work, I don't know which is best

**Review by iProgramStuff — APPROVED** [review]

**Pull request merged**